src/Playwright/Program.cs

```diff
@@ -39,6 +39,24 @@ public static int Main(string[] args)
 
         public int Run(string[] args)
         {
+            args ??= Array.Empty<string>();
+            for (int i = 0; i < args.Length; i++)
+            {
+                var a = args[i];
+                if (a.Contains(" "))
+                {
+                    var left = a.IndexOf('=');
+                    if (left >= 0)
+                    {
+                        args[i] = a.Substring(0, left + 1) + "\"" + a.Substring(left + 1, a.Length - left - 1) + "\"";
```

Comment:
what if value is already quoted, won't it end up double quoting it?

Reply:
.NET will end up stripping those quotes.